ReviewChecklist.md

```diff
@@ -34,6 +38,24 @@ It is also meant to serve as a final checklist for reviewers to go through befor
     - Review updated files before committing them.
         **Are expectations correct and do updated tests still serve their purpose?**
 
+## Abandoned PRs
+- [ ] **Is the submitter still responsive?**
+    - If the PR had no activity from the submitter in the last 2 weeks (despite receiving reviews and our prompts) we consider it abandoned.
```

Comment:
We did not agree on any specific time frame so this still up for discussion.

Reply:
It's 3 weeks for auto close, right? Stale label after 14 days, and then an auto-close after another 7 days, right @r0qs?

Reply:
Yes, but this actually does not apply to external PRs right now :)

I'll change to 3 weeks, though technically, it does not have to be the same. We could technically consider it abandoned right when it becomes stale even though it didn't get closed yet.

Reply:
Oh, don't bother doing that now, we can wait for next week so every can chime in.